grazer/src/main/webapp/WEB-INF/views/person.jsp

```diff
@@ -32,9 +32,11 @@
 
 <div>
 	<c:forEach var="listVar" items="${concept.getEqualTo() }"> 
-		<c:if test="${listVar != '[]'}">
-	    		<a href="${listVar}" target="_blank"> <i class="fa fa-external-link" aria-hidden="true"></i>       ${listVar} </a><br/>
-	    	</c:if>
+		<c:if test="${not empty listVar }">
+			<c:if test="${listVar != '[]'}">
```

Comment:
let's remove this test. The equalTo field does sometimes contain [] but it's a mistake and shouldn't  be there. This way we can see it and fix it in conceptpower.